libvast/src/system/spawn_index.cpp

```diff
@@ -30,18 +30,17 @@ spawn_index(node_actor::stateful_pointer<node_state> self,
   auto opt = [&](std::string_view key, auto default_value) {
     return get_or(args.inv.options, key, default_value);
   };
-  auto [archive, filesystem, accountant]
+  auto [filesystem, accountant, archive]
     = self->state.registry
-        .find<archive_actor, filesystem_actor, accountant_actor>();
-  if (!archive)
-    return caf::make_error(ec::lookup_error, "failed to find archive actor");
+        .find<filesystem_actor, accountant_actor, archive_actor>();
```

Comment:
nit: This is just reordering the list, would be nice to avoid such changes for a better `git blame` experience.